### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by NikolayGit1**
- [DEMO LINK](https://nikolaygit1.github.io/layout_html-form/)
- [TEST REPORT LINK](https://nikolaygit1.github.io/layout_html-form/report/html_report/)

### CODE REVIEW

File: .linthtmlrc.json [hunk 1]
```diff
@@ -1,3 +1,49 @@
 {
-  "extends": "@mate-academy/linthtml-config"
+  "attr-bans": [
```
Comment (vkryvytskyy) [inline .linthtmlrc.json:2]: https://user-images.githubusercontent.com/38065883/173183272-516e03a6-1d33-4210-8ee3-7d4c0696275e.mov

- there are missing colon after first label
- last input shouldn't have margin-bottom, you can see it from test

File: src/index.html [hunk 2]
```diff
@@ -2,16 +2,245 @@
...
+                name="surname"
+                id="surname"
+                autocomplete="off"
+                required
+              >
+            </label>
+          </div>
+          <div class="form-field">
+            <label for="name"
+              >Name:
```
Comment (vkryvytskyy) [inline src/index.html:36]:
```suggestion
            <label for="name">
              Name:
```

File: src/index.html [hunk 3]
```diff
@@ -2,16 +2,245 @@
...
+                max="100"
+                name="age"
+                id="age"
+                required
+              >
+            </label>
+          </div>
+          <div class="data">
+            <label for="fullAge"
+            >Full date of birth:
```
Comment (vkryvytskyy) [inline src/index.html:64]:
```suggestion
            <label for="fullAge">
              Full date of birth:
```

File: src/index.html [hunk 4]
```diff
@@ -2,16 +2,245 @@
...
+                type="date"
+                name="fullAge"
+                id="fullAge"
+                required
+              >
+            </label>
+          </div>
+          <div class="accept">
+            <label for="accept"
+            >I accept the term of the agreement
```
Comment (vkryvytskyy) [inline src/index.html:76]:
```suggestion
            <label for="accept">
              I accept the term of the agreement
```

File: src/index.html [hunk 5]
```diff
@@ -2,16 +2,245 @@
...
+            </label>
+          </div>
+        </div>
+      </fieldset>
+      <fieldset class="form-child">
+        <legend>Registration</legend>
+        <div class="block-2">
+          <div class="form-field-2">
+            <label for="email"
+            >E-mail:
```
Comment (vkryvytskyy) [inline src/index.html:93]:
```suggestion
            <label for="email">
              E-mail:
```

File: src/index.html [hunk 6]
```diff
@@ -2,16 +2,245 @@
...
+            <label for="cat">
+              <input
+                class="input"
+                type="radio"
+                name="cat"
+                value="Yes"
+                id="cat"
+                required
+              >
+        Yes
```
Comment (vkryvytskyy) [inline src/index.html:136]: Fix indentation

File: src/index.html [hunk 7]
```diff
@@ -2,16 +2,245 @@
...
+            </label>
+            <label for="cat1">
+              <input
+                class="input"
+                type="radio"
+                name="cat"
+                value="No"
+                id="cat1"
+              >
+        No
```
Comment (vkryvytskyy) [inline src/index.html:146]: Fix indentation

**Review by vkryvytskyy — CHANGES_REQUESTED** [review]

**Comment by NikolayGit1** [comment]
- [DEMO LINK](https://nikolaygit1.github.io/layout_html-form/)
- [TEST REPORT LINK](https://nikolaygit1.github.io/layout_html-form/report/html_report/)

**Pull request closed without merging**